packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -682,6 +684,59 @@ export class MockQuorumClients implements IQuorumClients, EventEmitter {
 	}
 }
 
+/**
+ * @alpha
+ */
+export interface IAudienceEvents extends IEvent {
```

Comment:
FYI - This is temporary, to reduce number of "borrowed" changes from another PR. All these changes will resolve to nothing when I merge the other PR (first).